pylint/utils/__init__.py

```diff
@@ -14,7 +14,6 @@
     HAS_ISORT_5,
     IsortDriver,
     _check_csv,
```

Comment:
I think we can remove it from this file and updated the docstring to be "Refactor the code to not use this old function".